ssl/quic/quic_port.c

```diff
@@ -875,9 +874,13 @@ static void port_send_retry(QUIC_PORT *port,
 
     /* Make space for integrity tag */
     token_buf_len += QUIC_RETRY_INTEGRITY_TAG_LEN;
-    token_buf = OPENSSL_realloc(token_buf, token_buf_len);
-    if (token_buf == NULL)
+    tmp_buf = OPENSSL_realloc(token_buf, token_buf_len);
```

Comment:
you can just factor this into your initial malloc.  The AES-GCM algorithm specified by the RFC always has an integrity tag length of 16 bytes.